assets/dev/scss/editor/panel/controls/_code.scss

```diff
@@ -0,0 +1,39 @@
+.elementor-control-type-code {
+
+	&.elementor-control-dynamic-value {
+
+		.elementor-control-dynamic-switcher {
+			border-radius: getValueByDirection(0 3px 3px 0, 3px 0 0 3px);
+			@include border-start-property(width, 0);
+		}
+	}
+
+	.elementor-control-dynamic-switcher {
+		border-radius: 0px 3px;
```

Comment:
Why you don't use `getValueByDirection` for this?

Reply:
It's like the same CSS on the textarea.